### ISSUE TITLE
Tests ran with the kotest plugin ignore unmock

### ISSUE THREAD

**Issue by matejdro** [kotest/kotest#5393]
[unmock](https://github.com/bjoernQ/unmock-plugin) is a gradle plugin that allows using android utility classes (such as Uri, Location, TextUtils etc.) within unit tests.

Unfortunately it does not seem to work with the kotest IntelliJ plugin, kotest seems to completely ignore unmock's existence.

Steps to reproduce:

1. Open [this project](https://github.com/kotest/kotest-intellij-plugin/files/10183457/Archive.zip) with android studio
2. Run `java/com/matejdro/kotestunmock/ExampleUnitTest.kt` with gradle 
3. Notice that test passes
4. Run "Given" block inside that test with the kotest plugin

BUG: Test will fail

**Comment by matejdro** [comment]
Possibly related to the https://github.com/kotest/kotest/issues/5413?

**Comment by matejdro** [comment]
This is still relevant and a major pain point - It prevents us from running individual tests of any code that includes utility Android classes, for example Location class.

**Comment by sksamuel** [comment]
Yeah kotest plugin doesn't use gradle, so anything setup there won't work.

**Comment by matejdro** [comment]
Yeah that's why I marked kotest/kotest#5413 as related - this issue is meant as another use case for running gradle instead of IntelliJ. But we can merge them if desired.

**Comment by LeoColman** [comment]
Let's close this issue for now, it's pretty stale :D

Please open a new ticket if you believe your issue is still unfixed
